### PR TITLE
Adding a blog to run Couchbase cluster on DC/OS

### PR DESCRIPTION
**Pull request by arun-gupta**
@ryadav88 FYI

## What are your changes?

New blog to run Couchbase cluster on DC/OS

## What is the urgency level of this change?
- [ ] Blocker
- [X] High
- [ ] Medium

## I have completed these items:
- [X] I have built locally and tested for broken links and formatting.
- [ ] If appropriate, I have added redirects.

### CODE REVIEW

**Comment by karlkfi** [comment]
@mesosphereci Test this please

**Comment by ryadav88** [comment]
@arun-gupta link validation seems to fail. Can you check your PR for broken links?

File: src/blog/2016-11-18_couchbase_cluster.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+For further information check out:
+
+* Couchbase [Developer Portal](http://developer.couchbase.com/server)
+* Ask questions on [Couchbase Forums](https://forums.couchbase.com/) or [Stack Overflow](http://stackoverflow.com/questions/tagged/couchbase)
+* Download [Couchbase](http://www.couchbase.com/nosql-databases/downloads)
+
+You can also follow us at [@couchbasedev](http://twitter.com/couchbasedev) and [@couchbase](http://twitter.com/couchbase).
+
+Many thanks to [@joerg_schad](https://twitter.com/joerg_schadhttp://www.couchbase.com/containers) for helping with understanding DC/OS concepts and debugging the configuration.
```
Comment (ryadav88) [inline src/blog/2016-11-18_couchbase_cluster.md]: should be `https://twitter.com/joerg_schad` instead

**Comment by emanic** [comment]
@mesosphereci Test this please

**Comment by ryadav88** [comment]
@mesosphere-ci Test this please

**Comment by emanic** [comment]
Can one of the admins verify this patch?

**Comment by karlkfi** [comment]
@mesosphere-ci Test this please

**Comment by karlkfi** [comment]
Sorry for the delay. Jenkins is configured to wait for admin approval to run the tests, to screen for malicious code. It's not super super obvious what the bot is waiting for, usually.

**Comment by karlkfi** [comment]
jwt.io has invalid certs, slack.arangodb.com was down for a while, and nagios.org seems to be offline. None of those are our problem today. You can ignore like link validation test failure.

**Pull request merged**